src/components/m-dropdown/_template.js

```diff
@@ -17,7 +17,7 @@ const nativeSelect = ({ title, items, size, value }) => html`<div class="${class
       items.map(({ name, value: itemValue, url }, index) => {
         const _itemValue = getItemValue(itemValue, index);
         return html`<option value="${_itemValue}" data-url="${url}" ${
-          _itemValue === value ? 'selected' : ''
+          `${_itemValue}` === `${value}` ? 'selected' : ''
```

Comment:
Guess this check should not be of strict type?
If so I would just write `_itemValue  == value`, the javascript engine will handle type conversion for you, so that stuff like `'4' == 4` will be truthy

Reply:
I can do that, will need to add  // eslint-disable-line though.

Reply:
NO PLEASE nogo for `_itemValue == value` from my side. Better type conversion to string so that we are type save. @nakovskijosif

Reply:
I will revert my last commit in a sec.

Reply:
I agree with you Luca, not many people read specs and definitely saver to stick with `===` than `==`. My point here is that this is perfect valid example of a `==` use case.

But to make this point more prevalent, we could also use `toString` conversion helpers, instead of meaning-lesl template lits

Reply:
ok with the toString. For me is the same as both clearly show what is happening.